keyboards/custommk/ergosentry_ansi/rules.mk

```diff
@@ -0,0 +1,5 @@
+SRC += matrix.c \
+       keyboards/custommk/cmk_cm6646.c
+
+CUSTOM_MATRIX = lite
+CIE1931_CURVE = yes
```

Comment:
If rgb matrix is enabled, this isn't needed, as rgb matrix forcibly enables this
```suggestion

```